README.md

```diff
@@ -232,41 +232,43 @@ git clone git@github.com:DazWilkin/gcp-exporter.git && cd gcp-exporter
 ### Usage
 
 ```bash
-gcp-exporter -h
+gcp-exporter --h
 
 Usage of gcp-exporter:
-  -collector.artifact_registry.disable
-    	Disables the metrics collector for the Artifact Registry
-  -collector.cloud_run.disable
-    	Disables the metrics collector for the Cloud Run
-  -collector.compute.disable
-    	Disables the metrics collector for the Compute
-  -collector.endpoints.disable
-    	Disables the metrics collector for the Cloud Endpoints Services
-  -collector.eventarc.disable
-    	Disables the metrics collector for the Cloud Eventarc
-  -collector.functions.disable
-    	Disables the metrics collector for the Cloud Functions
-  -collector.iam.disable
-    	Disables the metrics collector for the Cloud IAM
-  -collector.kubernetes.disable
-    	Disables the metrics collector for the Kubernetes (GKE)
-  -collector.logging.disable
-    	Disables the metrics collector for the Cloud Logging
-  -collector.monitoring.disable
-    	Disables the metrics collector for the Cloud Monitoring
-  -collector.scheduler.disable
-    	Disables the metrics collector for the Cloud Scheduler
-  -collector.storage.disable
-    	Disables the metrics collector for the Cloud Storage
-  -endpoint string
-    	The endpoint of the HTTP server (default ":9402")
-  -filter string
-    	Filter the results of the request
-  -max_projects int
-    	Maximum number of projects to include (default 10)
-  -path string
-    	The path on which Prometheus metrics will be served (default "/metrics")
+  --collector.artifact_registry.disable
+      Disables the metrics collector for the Artifact Registry
+  --collector.cloud_run.disable
+      Disables the metrics collector for Cloud Run
+  --collector.compute.disable
+      Disables the metrics collector for Compute Engine
+  --collector.endpoints.disable
+      Disables the metrics collector for Cloud Endpoints
+  --collector.eventarc.disable
+      Disables the metrics collector for Cloud Eventarc
+  --collector.functions.disable
+      Disables the metrics collector for Cloud Functions
+  --collector.gke.disable
+      Disables the metrics collector for Google Kubernetes Engine (GKE)
+  --collector.gke.infoMetric.enable
```

Comment:
Perhaps `--collector.gke.NodePoolMetrics.enable`?

Reply:
Not info metrics only for the Node Pool (it's info for both the control plane and the Node Pool).

What do you think about `collector.gke.ControlPlaneAndNodePoolInfoMetrics.enable`

Reply:
What about `collector.gke.ExtendedMetrics.enable`?